src/search/QuickOpenHelper.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2012 Adobe Systems Incorporated. All rights reserved.
```

Comment:
The new copyright notices has been changed to the form `2016 - present`. Please update the date in the format.